agent/exporter/src/main/java/com/microsoft/applicationinsights/agent/Exporter.java

```diff
@@ -72,12 +70,16 @@
 
     private static final Joiner JOINER = Joiner.on(", ");
 
-    private static final AttributeKey<Double> AI_SAMPLING_PERCENTAGE = AttributeKey.doubleKey("ai.internal.sampling.percentage");
+    private static final AttributeKey<Double> AI_SAMPLING_PERCENTAGE_KEY = AttributeKey.doubleKey("applicationinsights.internal.sampling_percentage");
```

Comment:
"sampling.percentage.key"?

Reply:
note that AI_SAMPLING_PERCENTAGE and AI_SAMPLING_PERCENTAGE_KEY have the same value.

Reply:
good point, I consolidated them